content/22.vaccines-trad.md

```diff
@@ -0,0 +1,507 @@
+Over the past 150 years, several approaches to vaccine development have been identified and refined [@doi:10.1073/pnas.1400472111].
+While the rapid release of the SARS-CoV-2 genome has been important in shaping the biomedical response to this pathogen, vaccines have been developed since long before the concept of a virus or viral genomes.
+As early as September 2020, there were over 180 vaccine candidates against SARS-CoV-2 in development [@doi:10.1038/s41586-020-2798-3].
+However, media attention has largely focused on vaccine development platforms that have produced vaccine candidates using previously unemployed technologies, especially mRNA vaccines.
+We review vaccine technologies used for SARS-CoV-2 in two parts: here, the application of traditional vaccine development platforms to SARS-CoV-2, and separately, omics-based approaches.
+
+Considering these well-established technologies is important for a global perspective on COVID-19 vaccines.
+As of {{owid_most_recent_date}}, {{viper_vaccine_counts}} SARS-CoV-2 vaccines have been approved world wide and {{owid_vaccine_counts}} are being administered throughout the world, with {{owid_total_vaccinations}} doses administered across {{owid_total_countries}} countries.
+Many vaccines are available in only a subset of countries, and the types of vaccines available varies widely throughout the world.
+Today, the requirements for developing and deploying a vaccine are complex and often require coordination between government, industry, academia, and philanthropic entities [@doi:10.1126/science.abc5312], and the Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
+While little is currently known about immunity to SARS-CoV-2, vaccine developers typically tests for serum neutralizing activity, as this has been established as a biomarker for adaptive immunity in other respiratory illnesses [@doi:10.1056/nejmoa2022483].
+However, unlike in efforts to develop vaccines for prior viral threats, the duration of the COVID-19 pandemic has made it possible to test vaccine in phase 3 trials where the effect of the vaccine on a cohort’s likelihood of contracting SARS-CoV-2 is evaluated.
+
+<!--To do: split so just the virion in is in this MS-->
+![
+**Structure of the SARS-CoV-2 virus.**
+The development of vaccines depends on the immune system recognizing the virus.
+Here, the structure of SARS-CoV-2 is represented both in the abstract and against a visualization of the virion.
+The abstracted visualization was made using BioRender [@url:https://biorender.com] and the microscopy was conducted by the National Institute of Allergy and Infectious Diseases [@url:https://www.niaid.nih.gov/news-events/novel-coronavirus-sarscov2-images].<!--To Do: Was a template used?-->
+](images/SARS_CoV_2.png){#fig:virus secno=1}
+
+<!--## Cross-Platform Considerations 
```

Comment:
```suggestion
<!--## Cross-Platform Considerations-->
```

Reply:
I'm taking a stab that this will change this will close the comment and make the other text appear darker?

Reply:
Ah I was trying to comment all this out because I have to figure out how to split it across the manuscripts! But you are 100% correct, that closes the comment!